### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by kimdv**
I think we should have a code of conduct.
So we just have a reference to Swift.org

Taken from https://github.com/apple/swift-docc/blob/main/CODE_OF_CONDUCT.md

### CODE REVIEW

**Comment by kimdv** [comment]
@swift-ci Please test

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Code of Conduct
+
+The code of conduct for this project can be found at https://swift.org/code-of-conduct.
+
+<!-- Copyright (c) 2021 Apple Inc and the Swift Project authors. All Rights Reserved. -->
```
Comment (ahoppen) [inline CODE_OF_CONDUCT.md:5]: Nitpick: 2022

**Review by ahoppen — APPROVED** [review]
Thanks

**Review by ahoppen — APPROVED** [review]

**Comment by kimdv** [comment]
@swift-ci Please test linux

**Pull request merged**